src/util/updateNote.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+export const updateNote = async (_id, value, token, noteDispatch) => {
+  try {
+    const options = { headers: { authorization: token } };
+    const response = await axios.post(
+      `/api/notes/${_id}`,
+      { note: { value } },
+      options
+    );
+    noteDispatch({ type: 'ADD_NEW_NOTE', payload: response.data.notes });
```

Comment:
check response status

Reply:
Will do the necessary changes thank you for pointing them out mohammed.